stories/components/Card.complexe.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
```

Comment:
Let's remove this, probably added for debugging?

Reply:
This is added for some click actions bellow in this file. This is require to avoid a warning, but we could also remove the `console.info`

Reply:
I see, let's leave it then 👍